sdk/personalizer/azure-ai-personalizer/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+## 1.0.0-beta.1 (2022-07-29)
+Version 1.0.0-beta.1 is a preview of our efforts in creating a Azure Personalizer client library that is developer-friendly
```

Comment:
This can be done in a follow up PR but we can mention some of the changes as here - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/personalizer/Azure.AI.Personalizer/CHANGELOG.md#200-beta1-2021-10-05

Reply:
Added a task for this in my product backlog item.